resources/agent.rb

```diff
@@ -24,18 +24,23 @@
 property :installer_url, String, default: lazy { "http://www.appveyor.com/downloads/deployment-agent/#{version}/AppveyorDeploymentAgent.msi" }
 property :install_path, String, default: lazy { 'C:\\Program Files (x86)\\AppVeyor\\DeploymentAgent\\Appveyor.DeploymentAgent.Service.exe' }
 
-action :create do
+default_action :install
+
+action :install do
   include_recipe 'windows'
 
-  package 'AppveyorDeploymentAgent' do
+  windows_package 'AppveyorDeploymentAgent' do
```

Comment:
Does this need to be windows package rather than just package? 
The package resource should detect it's on Windows.